DOCUMENTATION.md

```diff
@@ -0,0 +1,57 @@
+
+# Pages parameters
+
+
+The code handeling these paramaters is in `layouts/_default/single.html`
+
+## `untranslated: 1`
+
+When the website is build, a page with this parameter will:
+- Have it's content replaced by the English version
+- Links pointing to English pages that can be translated will be rewritten to points to the translation instead
+- The top of the page will have a message to encourage new translators
+
+
+## `do_not_translate: 1`
+
+A page with that attribute still had one version per language.
+
+That attribute has two functions:
+- It informs translators that this page should not be translated
+- The top of the page will **NOT** have a message to encourage new translators
+
+## `english_is_canonical: 1`
+
+A translation with that attribute will display a warning on the top to indicate that the English version is canonical.
+
+## `date` and `lastmod`
+
+Most pages displays the `lastmod` (or `date`) on the top. But the most important usage is to help syncronise translations: the `lastmod` of a translation must be equal to the `lastmod` of the translated page. So when a page is updated and need it's translations to be also updated, updating the `lastmod` helps to detect that automatically (on https://letsencrypt.org/i18n)
```

Comment:
```suggestion
Most pages displays the `lastmod` (or `date`) on the top. But the most important usage is to help synchronize translations: the `lastmod` of a translation must be equal to the `lastmod` of the translated page. So when a page is updated and need it's translations to be also updated, updating the `lastmod` helps to detect that automatically (on https://letsencrypt.org/i18n)
```

Note: "synchronise" is correct for British English; "synchronize" is US English. We generally use US spellings on the website.